### ISSUE TITLE
False positive on rgba fallback rule

### ISSUE THREAD

**Issue by shannonmoeller** [CSSLint/csslint#360]
I'm using a transparent png fallback for rgba background colors (a la Lea Verou's bulletproof, cross-browser RGBA backgrounds) which is generating the "Fallback background (hex or RGB) should precede RGBA background." error message.

``` css
background: url(../img/rgba-0_0_0_80.png);
background: rgba(0,0,0,.8);
```
